pokemon/convert.py

```diff
@@ -48,14 +42,11 @@ def convert_to_grayscale(image):
 def map_pixels_to_ascii_chars(image, range_width=25):
     """Maps each pixel to an ascii char based on the range
     in which it lies.
-
     0-255 is divided into 11 ranges of 25 pixels each.
     """
-
     pixels_in_image = list(image.getdata())
-    pixels_to_chars = [ASCII_CHARS[pixel_value/range_width] for pixel_value in
+    pixels_to_chars = [ASCII_CHARS[pixel_value//range_width] for pixel_value in
```

Comment:
Can you explain the change here?

Reply:
As I understand it, when using '/' in Python 3 it performs true division, so a float is returned, and this causes the error below.
```
Traceback (most recent call last):
  File ".\scripts\make_db.py", line 72, in <module>
    ascii = handle_image_conversion(image_file,new_width=30)
  File "C:\python38\lib\site-packages\pokemon-0.35-py3.8.egg\pokemon\convert.py", line 72, in handle_image_conversion
    image_ascii = convert_image_to_ascii(image,new_width)
  File "C:\python38\lib\site-packages\pokemon-0.35-py3.8.egg\pokemon\convert.py", line 56, in convert_image_to_ascii
    pixels_to_chars = map_pixels_to_ascii_chars(image)
  File "C:\python38\lib\site-packages\pokemon-0.35-py3.8.egg\pokemon\convert.py", line 48, in map_pixels_to_ascii_chars
    pixels_to_chars = [ASCII_CHARS[pixel_value/range_width] for pixel_value in
  File "C:\python38\lib\site-packages\pokemon-0.35-py3.8.egg\pokemon\convert.py", line 48, in <listcomp>
    pixels_to_chars = [ASCII_CHARS[pixel_value/range_width] for pixel_value in
TypeError: list indices must be integers or slices, not float
```
So I changed to '//' which does floor division. This [StackOverFlow](https://stackoverflow.com/a/13355867) where I found out about this.

Reply:
Ah, perfect! And that really reflects how old the library is - I wrote it originally with Python 2. I'll test this out locally to verify it's still working as expected.